projects/python/perception/panoptic_segmentation/efficient_lps/example_usage.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+from pathlib import Path
+
+from opendr.perception.panoptic_segmentation import EfficientLpsLearner, SemanticKittiDataset
+
+# User must download the datasets and set the paths accordingly
+DATA_ROOT = '/home/USER/data/efficientLPS'
+SEMANTIC_KITTI_ROOT = f'{DATA_ROOT}/converted_datasets/semantickitti_panoptic'
```

Comment:
does it have to be these locations? If not then perhaps then the data root should be passed instead by argument and default somewhere relative to the demo itself

Reply:
I have changed this into optional argument as suggested.